data/boundaries/citycountystate/index.html

```diff
@@ -88,6 +88,7 @@ <h5 id="MunicipalBoundaries-updates">Updates</h5>
         </div>
         <div class="panel-content">Updates to this dataset are published at a monthly interval or as needed.</p>
 <p>Recent update summaries:<br />
+<a href="{{site.baseurl}}{% post_url 2016-07-12-second-quarter-municipal-boundaries-updates.md %}">July 11, 2016</a><br />
```

Comment:
i fixed this for you but you do not put the .md part here see [these examples](https://github.com/agrc/agrc.github.io#common-website-tips) for tips on how to does this properly.